scripts/program/format.mjs

```diff
@@ -1,11 +1,18 @@
 #!/usr/bin/env zx
 import 'zx/globals';
-import { workingDirectory, getProgramFolders } from '../utils.mjs';
+import {
+  workingDirectory,
+  getRustfmtToolchain,
+  getProgramFolders,
+} from '../utils.mjs';
+
+const channel = getRustfmtToolchain();
+const toolchain = channel ? `+${channel}` : '';
```

Comment:
Yeah I know, but then I'd need to have two sets of functions - one with the `+` added, and one without it - so that the environment variables don't get the `+`. This felt like less pain.

I had:
- `getRustfmtToolchain()`
- `getRustfmtToolchainArg()`

...but hated it. 😅

Reply:
You can just have `getToolchainArg` as the generic function, and call `getToolchainArg(getRustfmtToolchain())`

Reply:
Haha I had this too, and scrapped it. I can add it back, though.